### PR TITLE
Lang_SChinese fully updated:)

### PR DESCRIPTION
**Pull request by FollowTheFate**
:)))))) I never use this before

### CODE REVIEW

**Comment by discus-sions** [comment]
That is the only issue I see.

File: lang_SChinese.yaml [hunk 1]
```diff
@@ -0,0 +1,1857 @@
...
+      UpdateText: 更新
+      UpdateAvailableText: 更新可用!
+    LotusButton: 莲花
+    InnerslothButton: 树懒
+  Factions:
+    Crewmates:
+      Name: Crewmates
+      Description: 完成任务&票出所有坏人.
+    Impostors:
+      Name: Impostors
```
Comment (discus-sions) [inline lang_SChinese.yaml:1812]: Impostors too.

File: lang_SChinese.yaml [hunk 2]
```diff
@@ -0,0 +1,1857 @@
...
+      Description: 击杀除了伪装者以外的所有玩家.
+    Madmates:
+      Name: Madmates
```
Comment (discus-sions) [inline lang_SChinese.yaml:1815]: Madmates too.

**Review by discus-sions — CHANGES_REQUESTED** [review]

File: lang_SChinese.yaml [hunk 3]
```diff
@@ -0,0 +1,1857 @@
...
+    ModUpdateMenu:
+      ExitGameText: 退出游戏
+      CloseText: 关闭
+      UpdateText: 更新
+      UpdateAvailableText: 更新可用!
+    LotusButton: 莲花
+    InnerslothButton: 树懒
+  Factions:
+    Crewmates:
+      Name: Crewmates
```
[the remark this answers is not in the document]
Comment (FollowTheFate) [reply lang_SChinese.yaml:1809]: Sorry!I didn't saw these:((

File: lang_SChinese.yaml (hunk 1 above)
Comment (FollowTheFate) [reply lang_SChinese.yaml:1812]: I will fix them:(((((

**Comment by FollowTheFate** [comment]
I hope there are no other issues:(

**Review by discus-sions — APPROVED** [review]
Thanks!

**Pull request merged**